src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -96,9 +97,15 @@
     "manufacturersXml": "../../../../src/app/zap-templates/zcl/data-model/manufacturers.xml",
     "options": {
         "text": {
-            "defaultResponsePolicy": ["Always", "Conditional", "Never"]
+            "defaultResponsePolicy": [
```

Comment:
Again, looks like you have some sort of auto-formatting turned on that disagrees with some existing auto-formatting.

Reply:
Corrected.